src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SupportApiServiceImpl.scala

```diff
@@ -0,0 +1,196 @@
+    val notBeforeConditions    = assertions.map(_.getConditions()).map(_.getNotBefore)
+    val notOnOrAfterConditions = assertions.map(_.getConditions()).map(_.getNotOnOrAfter())
+    val attributeStatements    = assertions.flatMap(_.getAttributeStatements().asScala.toList)
+    val attrs                  = attributeStatements.flatMap(_.getAttributes().asScala.toList)
+    val notBefore: List[OffsetDateTime] =
+      notBeforeConditions.map(dt => OffsetDateTime.ofInstant(Instant.ofEpochMilli(dt.getMillis()), ZoneOffset.UTC))
+    val notAfter: List[OffsetDateTime]  =
+      notOnOrAfterConditions.map(dt => OffsetDateTime.ofInstant(Instant.ofEpochMilli(dt.getMillis()), ZoneOffset.UTC))
+
+    val now = offsetDateTimeSupplier.get()
+
+    for {
+      _ <- Either
+        .catchNonFatal {
+          new SAMLSignatureProfileValidator().validate(
+            sig
+          ) // Indicates signature did not conform to SAML Signature profile
+        }
+        .void
+        .leftMap {
+          case e: ValidationException => SamlNotValid(e.getMessage())
+          case e                      => e
+        }
+        .toFuture
+      _ <- Future
+        .failed(SamlNotValid("Conditions NotBefore are not compliant"))
+        .whenA(notBefore.exists(nb => now.isBefore(nb)))
+      _ <- Future
+        .failed(SamlNotValid("Conditions NotOnOrAfter are not compliant"))
+        .whenA(notAfter.exists(na => now.isAfter(na)))
+      _ <- attrs
+        .find(a =>
+          a.getName == SUPPORT_LEVEL_NAME && (a
+            .getAttributeValues()
+            .asScala
+            .toList
+            .exists(av => SUPPORT_LEVELS.contains(av.getDOM().getTextContent())))
+        )
+        .toFuture(SamlNotValid("Support level is not compliant"))
+      _ <- Future
+        .failed(SamlNotValid("Conditions Audience are not compliant"))
+        .unlessA(
+          audienceRestriction
+            .flatMap(_.getAudiences().asScala.toList)
+            .exists(aud => ApplicationConfiguration.generatedJwtAudience.contains(aud.getAudienceURI))
+        )
+    } yield ()
+  }
+
+  private def buildClaims(selfcareId: String, tenant: TenantProcess.Tenant): Map[String, AnyRef] = {
+    import spray.json.RootJsonFormat
+    import spray.json._
+
+    case class Role(partyRole: String, role: String)
+    case class Organization(id: String, name: String, roles: Seq[Role])
+    implicit val roleFormat: RootJsonFormat[Role]                 = jsonFormat2(Role)
+    implicit val organizationFormat: RootJsonFormat[Organization] = jsonFormat3(Organization)
+
+    Map(
+      USER_ROLES            -> SUPPORT_ROLE,
+      ORGANIZATION_ID_CLAIM -> tenant.id.toString,
+      SELFCARE_ID_CLAIM     -> selfcareId,
+      ORGANIZATION          -> Organization(
+        id = selfcareId,
+        name = tenant.name,
+        roles = Seq(Role(partyRole = OPERATOR, role = SUPPORT_ROLE))
+      ).toJson.toString,
+      UID                   -> SUPPORT_ROLE
```

Comment:
This depends on what self-care sends to us, but UID could be a generic string and in this case, I suggest using SUPPORT to identify all calls coming from support.
The premise is that we can't log support people and moreover from Google App does not exist a UUID to identify them.

Reply:
point is that today some of our endpoints extract the UID as an UUID and they would fail.
I agree with you that it should be generic, but we need to align other services